### PR TITLE
Add digit buttons functionality

### LINKED ISSUE #15: Implement digit button functionality
Digit buttons should add the corresponding digit to the right of the number if it makes sense.

### PR DESCRIPTION
**Pull request by EdiOancea**
Closes #15

### CODE REVIEW

**Comment by EdiOancea** [comment]
@lgnstn updated

File: src/components/Calculator.js [hunk 1]
```diff
@@ -17,6 +17,23 @@ class Calculator extends Component {
     this.setState({ a: '0', b: '0', operation: '', displayed: '0' });
   }
   
+  addDigit(digit) {
+    const { a, b, operation } = this.state;
+    if(operation === '') {
```
Comment (lgnstn) [inline src/components/Calculator.js]: Missing space.

File: src/components/Calculator.js [hunk 2]
```diff
@@ -17,6 +17,23 @@ class Calculator extends Component {
...
+      if(a === '0') {
```
Comment (lgnstn) [inline src/components/Calculator.js]: Same.

File: src/components/Calculator.js [hunk 3]
```diff
@@ -17,6 +17,23 @@ class Calculator extends Component {
...
+        this.setState({ a: digit, displayed: digit });
+      } else if (a.length < 9) {
+        this.setState({ a: a.concat(digit), displayed: a.concat(digit) });
+      }
+    } else {
+      if(b === 0) {
```
Comment (lgnstn) [inline src/components/Calculator.js]: Same.

**Review by lgnstn — CHANGES_REQUESTED** [review]

**Pull request merged**